code/UserForm.php

```diff
@@ -326,7 +326,7 @@ public function getCMSFields()
             $fields->addFieldToTab('Root.Main', LiteralField::create(
                 'EmailRecipientsWarning',
                 '<p class="alert alert-warning">' . _t(
-                    __CLASS__.'.NORECIPIENTS',
+                    'SilverStripe\UserForms\Model\UserDefinedForm.NORECIPIENTS',
```

Comment:
```suggestion
                    'SilverStripe\\UserForms\\Model\\UserDefinedForm.NORECIPIENTS',
```

Reply:
@GuySartorelli ohh line break hit me - thx spotting this so timely!